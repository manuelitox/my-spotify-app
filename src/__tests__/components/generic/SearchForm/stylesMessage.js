import React from 'react'
import { shallow } from 'enzyme'

import StyledMessage from 'components/generic/SearchForm/stylesMessage'

describe('Component | StyledMessage', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <StyledMessage />
    )
  })
  it('should render a styled component', () => {
    expect(component).toMatchSnapshot()
  })
})