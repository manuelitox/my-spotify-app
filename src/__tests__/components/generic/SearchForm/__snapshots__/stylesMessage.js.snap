// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | StyledMessage should render a styled component 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  background-color: ",
          [Function],
          ";
  border-radius: 4px;
  color: ",
          [Function],
          ";
  font-size: 1rem;
  font-weight: 600;
  right: 14px;
  top: 16px;
  padding: 8px;
  position: absolute;
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
